pkg/proxy/ipvs/proxier.go

```diff
@@ -803,22 +803,20 @@ func cleanupIptablesLeftovers(ipt utiliptables.Interface) (encounteredError bool
 }
 
 // CleanupLeftovers clean up all ipvs and iptables rules created by ipvs Proxier.
-func CleanupLeftovers(ipvs utilipvs.Interface, ipt utiliptables.Interface, ipset utilipset.Interface, cleanupIPVS bool) (encounteredError bool) {
-	if cleanupIPVS {
-		// Return immediately when ipvs interface is nil - Probably initialization failed in somewhere.
-		if ipvs == nil {
-			return true
-		}
-		encounteredError = false
-		err := ipvs.Flush()
-		if err != nil {
-			klog.Errorf("Error flushing IPVS rules: %v", err)
-			encounteredError = true
-		}
+func CleanupLeftovers(ipvs utilipvs.Interface, ipt utiliptables.Interface, ipset utilipset.Interface) (encounteredError bool) {
+	// Return immediately when ipvs interface is nil - Probably initialization failed in somewhere.
+	if ipvs == nil {
+		return true
```

Comment:
if IPVS is nil we probably want to still run the iptables clean up, so maybe this should be:
```
if ipvs != nil {
    // IPVS clean up
}

// rest of the function..
```